recipes/ignition-utils/all/conanfile.py

```diff
@@ -0,0 +1,138 @@
+    def _minimum_cpp_standard(self):
+        return 17
+
+    @property
+    def _minimum_compilers_version(self):
+        return {
+            "Visual Studio": "16",
+            "gcc": "7",
+            "clang": "5",
+            "apple-clang": "10",
+        }
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def validate(self):
+        if self.settings.os == "Macos" and self.settings.arch == "armv8":
+            raise ConanInvalidConfiguration("sorry, M1 builds are not currently supported, give up!")
+    
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        if self.settings.compiler.cppstd:
+            tools.check_min_cppstd(self, self._minimum_cpp_standard)
+        min_version = self._minimum_compilers_version.get(str(self.settings.compiler))
+        if not min_version:
+            self.output.warn(
+                "{} recipe lacks information about the {} compiler support.".format(
+                    self.name, self.settings.compiler
+                )
+            )
+        else:
+            if tools.Version(self.settings.compiler.version) < min_version:
+                raise ConanInvalidConfiguration(
+                    "{} requires c++17 support. The current compiler {} {} does not support it.".format(
+                        self.name,
+                        self.settings.compiler,
+                        self.settings.compiler.version,
+                    )
+                )
+    
+    def requirements(self):
+        self.requires("cli11/2.1.2")
+        self.requires("doxygen/1.9.2")
+
+    def build_requirements(self):
+        self.build_requires("ignition-cmake/2.10.0")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        version_major = self.version.split(".")[0]
+        conan.tools.files.rename(
+             self, "ign-utils-ignition-utils{}_{}".format(version_major, self.version),
+             self._source_subfolder
+            )
+
+    def _configure_cmake(self):
+        if self._cmake is not None:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.definitions["BUILD_TESTING"] = False
+        #self._cmake.definitions["IGN_UTILS_VENDOR_CLI11"] = True
+        self._cmake.definitions["CMAKE_FIND_DEBUG_MODE"] = "1"
+        self._cmake.configure()
+        return self._cmake
+
+    def build(self):
+        for patch in self.conan_data.get("patches", {}).get(self.version, []):
+            tools.patch(**patch)
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def package(self):
+        self.copy("LICENSE", dst="licenses", src=self._source_subfolder)
+        cli_header_src = os.path.join(self._source_subfolder, "cli", "include")
+        if int(tools.Version(self.version).minor) is 0:
```

Comment:
```suggestion
        if int(tools.Version(self.version).minor) == 0:
```
`is` should be used only for reference equality